docs/source/additional-resources/community-edition/community-edition.rst

```diff
@@ -0,0 +1,105 @@
+:orphan:
+
+.. _community-edition:
+
+Use Databricks Community Edition
+================================
+
+Try Glow on Databricks for free with `Community Edition <https://databricks.com/product/faq/community-edition>`_.
```

Comment:
I would use Databricks Community Edition everywhere instead of Community Edition.

Reply:
Good call, thanks!